figs/species_fig.R

```diff
@@ -225,14 +202,9 @@ tree_plot +
              alpha=.7
   ) +
   scale_color_manual(name="count", na.value="white", values = c("grey","black")) +
-  # scale_x_discrete(position="top") +
   ggplot2::xlim(0, 8000) +
   ggplot2::ylim(-1.5,21.5)
 
 ggsave("./species_fig_temp6.png",height=9,width=9)
 
 # increase size of column labels, angle, remove "NA" in legend - do this outside R because ggtree doesn't want to
-=======
-dev.off()
```

Comment:
This is necessary to close the device opened by the call to `pdf( )` way up there... but, I'm not sure how the figure's being made any more?